scripts/setup_host.sh

```diff
@@ -196,21 +196,20 @@ function ubu_enable_host_sriov(){
 }
 
 function ubu_update_fw(){
-    FW_REL="linux-firmware-20220310"
-    GUC_REL="70.0.3"
-    HUC_REL="7.9.3"
-    S_DMC_REL="2_01"
-    P_DMC_REL="2_12"
+    FW_REL="linux-firmware-20221109"
+    GUC_REL="70"
+    TGL_GUC_REL="70.1.1"
```

Comment:
why is it specified as TGL? what about other platform such as ADL?

Reply:
The adl load the adlp_guc_70.bin, adlp_dmc_ver2_16.bin, tgl_huc.bin.
Is included,
GUC_REL="70"
adlp_guc_$GUC_REL.bin tgl_guc_$GUC_REL.bin dg2_guc_$GUC_REL.bin
The specified TGL_GUC is used on tgl, i think is better to compatible TGL, so i add them. Is that ok?